portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase

```diff
@@ -46,12 +46,52 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8115 | Test Summary: Assert document list shows all of the documents and media assets in a list"
-	@ignore = "true"
 	@priority = "5"
 	test ViewAllDocumentsAndMediaShownInAssetList {
+		for (var layoutName : list "AC Page 2,AC Page 3,AC Page 4") {
+			ACUtils.addSiteAndPage(layoutName = "${layoutName}");
+		}
+
+		for (var n : list "2,3,4") {
+			ACUtils.createDMAndAddToPage(
+				dmDocumentFile = "Document_${n}.jpg",
+				dmDocumentTitle = "AC Page ${n}",
+				layoutName = "AC Page ${n}");
+		}
+
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		for (var n : list "1,2,3,4") {
+			for (var pageName : list "AC Page 2,AC Page 3,AC Page 4") {
+				Navigator.gotoSitePage(
+					pageName = "${pageName}",
+					siteName = "Site Name");
+
+				DMNavigator.gotoDocumentCP(dmDocumentTitle = "${pageName}");
+			}
+		}
+
+		ACUtils.closeAllSessions();
+
+		Pause(locator1 = "30000");
+
+		ACUtils.launchAC();
 
-		// TODO LRAC-8115 ViewAllDocumentsAndMediaShownInAssetList pending implementation
+		ACProperties.switchProperty(propertyName = "${propertyName}");
 
+		ACNavigation.goToAssets();
+
+		ACNavigation.switchTab(tabName = "Documents and Media");
+
+		ACTimeFilter.clickTimeFilterButton();
+
+		ACTimeFilter.setLast24Hours();
+
+		for (var pageName : list "AC Page 2,AC Page 3,AC Page 4") {
```

Comment:
Hey, @AfonsoHSantos. I think you forgot to validate the **AC Page** that is created in the test setUp.